rurusetto/users/templates/users/settings.html

```diff
@@ -229,11 +229,11 @@ <h2>Support</h2>
 <div class="container">
     <a id="account"><h1>Account</h1></a>
     <h2>Account Connection</h2>
-    <p class="raleway-font">You can connect your account with your osu! account so you can login by using your osu! account instead of username and password.</p>
+    <p class="raleway-font">You can connect your account with your osu! account so you can login by using your osu! account instead of Rurusetto username and password.</p>
```

Comment:
```suggestion
    <p class="raleway-font">You can connect your account with your osu! account so you can login by using your osu! account instead of Rūrusetto username and password.</p>
```
A little bit typo on the name.